api.ts

```diff
@@ -755,7 +755,7 @@ export interface SyncFormulaDef<
    * the new state of each object.
    */
   /** @hidden */
-   executeUpdate?(
+  executeUpdate?(
```

Comment:
I noticed an extra space crept in here, so just cleaning that up while I'm at it.